frontend/package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@chakra-ui/icons": "^1.0.16",
```

Comment:
Do we need this icons if we have font awesome?

Reply:
I used chakra ui icons as I don't have font awesome installed. This should stay as is and you'll replace chakra ui icons with font awesome icons and remove the dependency as part of your ticket.